test/test_validators.py

```diff
@@ -98,11 +97,15 @@ def mock_doc(sites_of_involvement):
         self.entities.append(mock_doc(["Cervix", "Ovary, NOS"]))
 
         self.json_validator.record_errors(self.entities)
-        self.assertEqual(1, len(self.entities[0].errors))
-        self.assertEqual(1, len(self.entities[1].errors))
-        self.assertEqual(0, len(self.entities[2].errors))
-        self.assertEqual(["sites_of_involvement.0"], self.entities[0].errors[0]["keys"])
-        self.assertEqual(["sites_of_involvement.1"], self.entities[1].errors[0]["keys"])
+        self.assertEqual(2, len(self.entities[0].errors))
+        self.assertEqual(2, len(self.entities[1].errors))
+        self.assertEqual(1, len(self.entities[2].errors))
+        error_keys_zero = [e["keys"][0] for e in self.entities[0].errors]
+        error_keys_one = [e["keys"][0] for e in self.entities[1].errors]
+        assert "sites_of_involvement.0" in error_keys_zero
+        assert "diagnosis_is_primary_disease" in error_keys_zero
+        assert "sites_of_involvement.1" in error_keys_one
+        assert "diagnosis_is_primary_disease" in error_keys_one
```

Comment:
since we are in a test case:
```suggestion
        self.assertIn("sites_of_involvement.0", error_keys_zero)
        self.assertIn("diagnosis_is_primary_disease", error_keys_zero)
        self.assertIn("sites_of_involvement.1", error_keys_one)
        self.assertIn("diagnosis_is_primary_disease",error_keys_one)
```